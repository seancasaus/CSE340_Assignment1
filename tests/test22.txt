GFmP6 ((V)|(v)).(((s)*)|((S).(c))) , jThyL S , MR1S6 (((V)|(Z))*).((V)|(Z)) , T1 ((K).(X)).(e) #
"SVSZZKXe KXeZGSKXe SKXevSc " 
