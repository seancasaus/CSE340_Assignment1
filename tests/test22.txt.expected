jThyL , "S"
GFmP6 , "V"
jThyL , "S"
MR1S6 , "ZZ"
T1 , "KXe"
T1 , "KXe"
MR1S6 , "Z"
ERROR
